apiary.apib

```diff
@@ -99,8 +99,29 @@ TODO
 # Group cat
 
 ## cat [GET /cat]
+Retrieves the object named by <ipfs-or-ipns-path> and outputs the data it contains.
 
-+ Response 200 (text/plain)
++ Parameters
+    + arg (string, required) - The path to the IPFS object(s) to be outputted
+
++ Response 200
+
+    + Headers
+        - Transfer-Encoding: chunked
+        - X-Stream-Output: 1
+
+    + Body
+
+    ```
+    $ curl -i http://localhost:5001/api/v0/cat?arg=QmeomffUNfmQy76CQGy9NdmqEnnHU9soCexBnGU3ezPHVH
```

Comment:
Yeah. Not sure how to show the file we're using as the test? This is wrong; looking for a solution now.